File: 03_recursion/ES6/02_greet.js
const greet2 = name => console.log(`how are you, ${name}?`);

const bye = () => console.log('ok bye!');

const greet = (name) => {
  console.log(`hello, ${name}!`);
  greet2(name);
  console.log('getting ready to say bye...');
  bye();
};

greet('adit');

File: 03_recursion/perl5/02_greet.pl
sub greet2 {

    my $name = shift;

    print "how are you, $name ?\n";
}

sub bye { print "ok bye !\n"; }

sub greet {

    my $name = shift;

    print "hello, $name !\n";

    greet2($name);

    print "getting ready to say bye...\n";

    bye();
}

greet('adit');
